tests/expected/arith_checks/fixme_check_arith_failures.rs

```diff
@@ -0,0 +1,13 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+//! Check that code after an overflow check failure is unreachable.
+
+#[cfg_attr(kani, kani::proof)]
```

Comment:
Is this to allow us to run it with `rustc`? (Neat!)

Reply:
Yeah, but I can remove it. I use that a lot when comparing the two but sometimes I forget to remove them.